docs/overview.md

```diff
@@ -1,44 +1,41 @@
 # Overview
 
-## What is `jrnl`?
-
 `jrnl` is a simple but powerful plain text journal application for the command
 line.
 
-`jrnl` is not the only plain text journal application. It includes features that
-are common to many other applications:
-
-- Journal entries are created and displayed directly from the command line.
-- Journals are stored as human-readable, future-proof plain text files. Because
-  files are stored as plain text, you can rest assured that your journals will
-  still be readable in 2050 and beyond. Entries can be searched, filtered, and
-  edited rapidly and easily.
-- Journals can be stored anywhere, including in shared folders so that they can
-  be synchronized between devices.
-- Journal entries can be created and modified using an external editor of your
-  choice, using the command line as a starting point to filter specific entries.
-- BeJournals are stored in single files using date/time tags to identify
-  individual entries. Some systems endorse a file/folder methodology; `jrnl`
-  keeps 
-- Journals can be protected using [256-bit
-  AES encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
-
-## Why keep a journal?
-
-Journals aren't just for people who have too much time on their summer vacation.
-Journals help you keep track of events, thoughts, feelings, tasks, ideas, and
-anything else that is worth remembering. Your imagination may be limitless, but
-your memory probably isn't.
-
-To begin, try to write at least 20 words a day. `jrnl` makes this incredibly
-easy. Fire up a terminal and type, `jrnl today at 8am: I arrived at work to find
-a birthday present on my desk. My colleagues are the best!` You can reflect on
-what made your day special so that you'll always remember why you haven't wasted
-it. Or try using `jrnl` to keep a gratitude journal, a habit that has been shown
-to improve mental health and quality of life.
-
-For professional use, a text-based journal can be the perfect complement to your
-to-do list -- a record of what you've done as well as how and when you've done
-it. You can use a journal as a quick way to keep a change log (`jrnl Updated
-jrnl overview page to make it more awesomer`). Or use it to keep a lab notebook.
-The possibilities are almost endless.
+Your imagination may be limitless, but your memory probably isn't. `jrnl` lets
+you write something down, and then it gets out of the way.
+
+## how it works
```

Comment:
I think these should be capital, but this is obviously just an aesthetic choice.

Reply:
Like, 'How It Works,' or 'HOW IT WORKS'? The first way is the most traditional.

Reply:
First one imo

Reply:
All headings should probably be fixed. Look up "titlecase" if you want a deep dive into what words to capitalize and which ones not to.